src/Magento/FunctionalTestingFramework/DataGenerator/Handlers/CredentialStore.php

```diff
@@ -9,12 +9,18 @@
 use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
 use Magento\FunctionalTestingFramework\DataGenerator\Handlers\SecretStorage\FileStorage;
 use Magento\FunctionalTestingFramework\DataGenerator\Handlers\SecretStorage\VaultStorage;
+use Magento\FunctionalTestingFramework\DataGenerator\Handlers\SecretStorage\AwsSecretManagerStorage;
 use Magento\FunctionalTestingFramework\Util\Path\UrlFormatter;
 
 class CredentialStore
```

Comment:
there are some comments for `getSecret` that only talks about vault and file storage, needs to be reworded to include `AWS secrets manager`

> / Get secret data from storage according to the order they are stored
        // File storage is preferred over vault storage to allow local secret value overriding remote secret value

Reply:
Fixed